formatter-openapi/src/main/scala/pl/iterators/baklava/formatter/openapi/OpenApiFormatterWorker.scala

```diff
@@ -99,12 +100,18 @@ class OpenApiFormatterWorker(jsonSchemaToSwaggerSchemaWorker: JsonSchemaToSwagge
 
   private def queryParamsToParams(parameters: List[RouteParameterRepresentation[_]]): List[Parameter] = {
     parameters.map { param =>
+      val schema = new StringSchema
+      schema.setExample(param.sampleValue)
+      param.enums.foreach { values =>
+        schema.setEnum(values.toList.asJava)
+      }
+
```

Comment:
Work nice to enum values, but example is still not rendered correctly:
![Screenshot from 2023-10-18 11-19-58](https://github.com/theiterators/baklava/assets/355631/3f71ba84-80e8-4699-8957-3238782c799e)

Reply:
I example in the param is rendered incorrectly, while in the schema it is okay:
```
      - name: role
        in: query
        required: false
        schema:
          type: string
          example: Admin
          enum:
          - Admin
        example: {}
```